test/dispose.js

```diff
@@ -0,0 +1,42 @@
+describe('using the watcher API to dispose and watch again', function() {
```

Comment:
quick question, did you run the test? Do you know how to run them? seems like the test is broken :)